### PR TITLE
release 0.6.0 (#451)

### LINKED ISSUE #451: Release 0.6.0
With the new feature that allows you to follow logs in #215 we should release a new binary.

### PR DESCRIPTION
**Pull request by adietish**
fixes #451

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]
LGTM, worth mentioning the follow logs action in the README?

**Comment by adietish** [comment]
> LGTM, worth mentioning the follow logs action in the README?

Excellent point, will add a section to the README

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
added entries to the README, waiting for @mohitsuman to review my english

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ It is available for install from the [JetBrains Marketplace](https://plugins.jet
  - Support Creation and Deletion of new Kubernetes resources on Cluster
  - Navigate into Kubernetes workloads, services, pods and nodes
  - Switching contexts and namespaces within Kubernetes
+ - Follow logs from your clusters
```
Comment (mohitsuman) [inline README.md:20]:
```suggestion
 - Follow logs from the connected cluster resource
```

File: README.md [hunk 2]
```diff
@@ -111,6 +112,13 @@ The very same dialog may also be opened with a link that's present in the notifi
 #### Delete Resources
 You may delete any resource that is listed in the resource tree by choosing "Delete" in the context menu.
 
+#### Follow logs
+
+Intellij-Kubernetes allows you to follow the logs of Pods, Jobs and Builds. 
+A dedicated tab continuously displays the messages that the chosen resource sends to its log.
```
Comment (mohitsuman) [inline README.md:118]:
```suggestion
There is a dedicated tab to display the log messages for the selected Kubernetes resource.
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]

**Comment by adietish** [comment]
@mohitsuman thx for the improvements!

File: README.md [hunk 3]
```diff
@@ -111,6 +112,13 @@ The very same dialog may also be opened with a link that's present in the notifi
 #### Delete Resources
 You may delete any resource that is listed in the resource tree by choosing "Delete" in the context menu.
 
+#### Follow logs
+
+Intellij-Kubernetes allows you to follow the logs of Pods, Jobs and Builds.
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply README.md]: extension is for VSCode, plugin is to be used here

**Pull request merged**